vision/README.md

```diff
@@ -90,14 +90,14 @@ to [browse the source][google-cloud-java-source] and
 [report issues][google-cloud-java-issues].
 
 ## Troubleshooting
+ * **"Failed to detect whether we are running on Google ComputeEngine"**; **"The Application Default Credentials are not available"**: If you get these errors, make sure you set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to a path to your service account credentials. If you don't already have service account credentials, you will need to
+[create them][create-creds]. 
 
- * **"Failed to detect whether we are running on Google ComputeEngine"**
- * **"The Application Default Credentials are not available"**
+   **Note: Sometimes the system will only recognize the newly set environment variable after you restart your computer.**
```

Comment:
This is not true. Environment variables do not require a computer restart

Reply:
Unfortunately, it is true on Windows, at least when I tested.

Reply:
I still do not believe this is true. You need to restart your prompt is all. Summoning @SurferJeffAtGoogle to weigh in.

Reply:
Okay

Reply:
https://serverfault.com/questions/8855/how-do-you-add-a-windows-environment-variable-without-rebooting

Reply:
Please remove this line and we can merge this pull request!

Reply:
Sure!